integ-test/src/test/java/org/opensearch/sql/util/TestUtils.java

```diff
@@ -772,6 +779,29 @@ public static String getResponseBody(Response response, boolean retainNewLines)
     return sb.toString();
   }
 
+  // TODO: this is temporary fix for fixing serverless tests to pass with 2 digit precision value
```

Comment:
any thoughts on what should be long term fix for this and other temp fixes? Also, should we create issues to track long term fix?